_episodes/02-snakefiles.md

```diff
@@ -81,11 +81,15 @@ By default, Snakemake looks for a file called `Snakefile`, and we can run
 Snakemake as follows:
 
 ~~~
-snakemake
+snakemake --cores 1
 ~~~
 {: .language-bash}
 
-By default, Snakemake tells us what it's doing as it executes actions:
+The number of cores to use is a required argument for versions 5.11 and 
+above of Snakemake.  It is possible to use more cores, but a single core 
```

Comment:
```suggestion
The number of cores to use (`--cores`) is a required argument for recent versions
of Snakemake.  It is possible to use more cores, but a single core 
```

Reply:
It may be helpful to specify versions explicitly. Features in some Python packages can change rapidly,

Reply:
In my opinion, as Snakemake is currently at v7.x, you might even get rid of the mention of versions and say "... is a required argument." But perhaps there are a lot of people running old versions of the tool for some reason I am not aware of. 

However, I really do not feel strongly about this change, so feel free to ignore/reject it.

Reply:
Will leave that as is for now. Hopefully it will be updated in a year or so.